daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-discover-services.md

```diff
@@ -403,21 +404,25 @@ dapr invoke --app-id checkout --method checkout/100
 
 When running on [namespace supported platforms]({{< ref "service_invocation_api.md#namespace-supported-platforms" >}}), you include the namespace of the target app in the app ID: `checkout.production`
 
-For example, invoking the example service with a namespace would be:
+For example, invoking the example service with a namespace would look like:
 
 ```bash
 curl http://localhost:3602/v1.0/invoke/checkout.production/method/checkout/100 -X POST
 ```
 
 See the [Cross namespace API spec]({{< ref "service_invocation_api.md#cross-namespace-invocation" >}}) for more information on namespaces.
 
-## Step 3: View traces and logs
+## View traces and logs
+
+Our example above showed you how to directly invoke a different service running locally or in Kubernetes. Dapr:
 
-The example above showed you how to directly invoke a different service running locally or in Kubernetes. Dapr outputs metrics, tracing and logging information allowing you to visualize a call graph between services, log errors and optionally log the payload body.
+- Outputs metrics, tracing, and logging information,
+- Allows you to visualize a call graph between services and log errors, and
+- Optionally, log the payload body.
 
-For more information on tracing and logs see the [observability]({{< ref observability-concept.md >}}) article.
+For more information on tracing and logs, see the [observability]({{< ref observability-concept.md >}}) article.
 
- ## Related Links
+## Related Links
 
-* [Service invocation overview]({{< ref service-invocation-overview.md >}})
-* [Service invocation API specification]({{< ref service_invocation_api.md >}})
+- [Service invocation overview]({{< ref service-invocation-overview.md >}})
+- [Service invocation API specification]({{< ref service_invocation_api.md >}})
```

Comment:
How should Quickstarts or tutorials be in these HowTos? Should they be a reference, or is that overkill if they are already in the overview?